_data/taps/versions/bing-ads.yml

```diff
@@ -0,0 +1,161 @@
+# -------------------------- #
+#      BING ADS VERSIONS     #
+# -------------------------- #
+
+latest-version: "2.0"
+
+released-versions:
+  - number: "2.0"
+    date-released: "October 4, 2018"
+    # date-last-connection:
+    deprecated: false
+    deprecation-date: "n/a"
+
+  - number: "1.0"
+    date-released: ""
```

Comment:
The 1.0.0 release went out on `2018-05-04`, if you want to add that here.

Reply:
Thanks! Added